### DISCUSSION TITLE
Help My g-helper doesn't show power limit Please help

### DISCUSSION THREAD

**Discussion by JinDojun** [seerge/g-helper#2835] [Q&A]
![image](https://github.com/user-attachments/assets/eec46df1-dc91-4771-ae83-185dc7dcd98b)
 
Laptop: Asus Tuf Gaming f15 FX506HC ( Intel i5-11400H+ RTX3050).
G-Helper Version 0.179.0 (Lastest)
Under latest BIOS v311

![image](https://github.com/user-attachments/assets/e563f231-b93f-4aef-a5af-ee6f58a35841)

**Comment by seerge** [comment]
@JinDojun hello, power limits need to be supported by hardware / BIOS. 

You never had power limits in Armoury, so you won't have them here :) It's normal.

The only way to (indirectly) limit power for your CPU - is change CPU boost or disable it completely for your case

**Reply by JinDojun** [reply]
Oh Thank u so much. i use first time this i unstall Armoury because laggy. Thank again my eng not good

**Reply by JinDojun** [reply]
Would u show setting. when unplug batteries im lag little bit. u recomended me @seerge

**Comment by KuruSa-arch** [comment]
did u find a way? i got the same model

**Reply by seerge** [reply]
Same model - same answer :) it doesn't support power limits in an "armory-way" so they are not available here. You can try Throttlestop tho if disabling Boost is not enough for you

**Reply by KuruSa-arch** [reply]
alright sir thank you nevertheless
